schema/src/PrimaryImpact.yaml

```diff
@@ -0,0 +1,34 @@
+id: https://framework.regen.network/schema/PrimaryImpact
+name: PrimaryImpact
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+  qudt: http://qudt.org/schema/qudt/
+  unit: http://qudt.org/vocab/unit/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - SDG
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  PrimaryImpact:
+    class_uri: rfs:PrimaryImpact
+    title: Primary Impact
+    description: Primary impact of the biodiversity project.
+    slots:
+      - sdg
+    attributes:
+      id:
+        identifier: true
```

Comment:
Some of these classes have `id` attributes, I believe from @S4mmyb initial migration trying to match the existing schemas. I added the `identifier` property so that linkml treats it as such. But I'm curious if we actually want to have full identifiers impacts, benefits, etc? If we are using identifiers does that mean we actually should be pulling from a taxonomy and have a semantic enum?

I need to review existing data more, just leaving the comment while its on my mind cc @clevinson

Reply:
I don't think it necessarily means that we need a semantic enum, but it should be something that is a full URI.

From the LinkML [docs](https://linkml.io/linkml/schemas/slots.html#identifiers):

> the range of an identifier can be any type, but it is a good idea to have these be of type [Uriorcurie](https://w3id.org/linkml/Uriorcurie)
> 
> A class must not have more than one identifier (asserted or derived). identifier marks the primary identifier.

I'm fine with us using app.regen.network/credit-class/C01 , or `http://mainnet.regen.network:1317/regen/ecocredit/v1/class/C01` for these, or some other resource list that we can work on making fully resolvable later (e.g. registry.regen.network/C01 for any arbitrary resource that exists on-chain)